packages/sharing-dialog/src/SharingDialog.component.js

```diff
@@ -5,21 +5,27 @@ import DialogContent from '@material-ui/core/DialogContent';
 import DialogActions from '@material-ui/core/DialogActions';
 import Button from '@material-ui/core/Button';
 import Snackbar from '@material-ui/core/Snackbar';
+import CircularProgress from '@material-ui/core/CircularProgress';
 import PropTypes from 'prop-types';
 import Sharing from './Sharing.component';
-import { LoadingMask } from '@dhis2/d2-ui-core';
-
-const styles = {
-    loadingMask: {
-        position: 'relative',
-    },
-};
 
 const defaultState = {
     sharedObject: null,
     errorMessage: '',
 };
 
+const loadingMaskStyle = {
+    position: 'relative',
+    left: '45%',
+    top: '45%',
+};
+
+const LoadingMask = () => (
+    <div style={loadingMaskStyle} >
```

Comment:
Do we prefer `style` or `className` here?  Usually I would prefer withStyles() and className but I don't know if we've standardized on that.

Reply:
As far as I know, we have not standardized, though I agree with your preferences.